glide.lock

```diff
@@ -1,5 +1,5 @@
 hash: fff401d3955c816110993c628fa41dd5f983036587ee137479f478b3e967dccb
-updated: 2017-04-26T21:39:21.015444832Z
+updated: 2017-04-26T23:59:00.032458759Z
```

Comment:
ok, i'll create a new pr with just this vendor update and update this pr to remove this vendor update, but it'll needed the pr vendor update merged first before be merged